vm/interpreter/src/vm.rs

```diff
@@ -83,14 +83,14 @@ pub struct VM<DB: BlockStore + 'static, P = DefaultNetworkParams> {
 
 impl<DB, P> VM<DB, P>
 where
-    DB: BlockStore,
+    DB: BlockStore + Clone,
     P: NetworkParams,
 {
     #[allow(clippy::too_many_arguments)]
     pub fn new<R, C, LB>(
         root: Cid,
         store: &DB,
-        store_arc: Arc<DB>,
+        store_arc: DB,
```

Comment:
We can remove the `store` argument and just use `store_arc` instead.

Reply:
Removed